modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/UserGroupResourceTest.java

```diff
@@ -77,6 +104,28 @@ public void testGetUserUserGroups() throws Exception {
 		assertValid(page);
 	}
 
+	@Override
+	@Test
+	public void testPostUserGroupByExternalReferenceCodeUsers()
+		throws Exception {
+
+		UserGroup userGroup =
+			testPostUserGroupByExternalReferenceCodeUsers_addUserGroup();
+
+		Assert.assertEquals(0, (int)userGroup.getUsersCount());
```

Comment:
why is the cast to int needed?

Reply:
because the getUsersCount returns a wrapper object Integer. 
Should it be Assert.assertEquals(Integer.valueOf(0), userGroup.getUsersCount()) instead?